README.md

```diff
@@ -1,106 +1,125 @@
-*Looking for the [Dwellingly App front end?](https://github.com/codeforpdx/dwellingly-app)*
+_Looking for the [Dwellingly App front end?](https://github.com/codeforpdx/dwellingly-app)_
 
 ## Dwellingly App Backend
 
 Set up Dwelling Flask Testing Backend (for the first time)
-NOTE: Database is SQLite3 via SQLAlchemy 
+NOTE: Database is SQLite3 via SQLAlchemy
 
-+ Github Repo: { https://github.com/codeforpdx/dwellinglybackend }
-+ Live Server: {https://dwellinglyapi.herokuapp.com/} 
+- Github Repo: { https://github.com/codeforpdx/dwellinglybackend }
+- Live Server: {https://dwellinglyapi.herokuapp.com/}
 
 ### To Start Server
 
 1. Clone the repo (`git clone https://github.com/codeforpdx/dwellinglybackend.git`)
 2. Install Python ( https://realpython.com/installing-python/ )
-3. Install pipenv: `pip3 install --user pipenv`
+3. Install pipenv: `pip3 install --user pipenv`.
 4. Seed the database
-    - Run: `python seed_db.py`
-      - To re-seed the database from scratch, delete data.db before running the script
-    - Look for the file data.db to be created in the root directory
 
-5. Start the server using the flask environment (required every time the project is re-opened):
-    - Run: `pipenv run flask run`
-    - Run + restart the server on changes:  `pipenv run flask run --reload`
+   - Run: `python seed_db.py`
+     - To re-seed the database from scratch, delete data.db before running the script
+   - Look for the file data.db to be created in the root directory
+
+5. Run `pipenv install`
```

Comment:
I think `pipenv install` should come before `python seed_db.py`. It may not matter now but if in the future the `seed_db.py` script requires any dependencies, then it would become required

Reply:
_yes_ it does need to come first as we just saw in a meeting 😄